ansible/playbooks/nuke.yml

```diff
@@ -39,3 +39,12 @@
       file:
         path: /etc/cni/net.d
         state: absent
+
+    - name: remove apt-mark hold
```

Comment:
Add a `when` condition to only do this on debian based distro's.

Reply:
after this, LGTM!

Reply:
Fixed - I think the ansible-lint errors should be addressed perhaps on a separate PR.

Reply:
Agreed.